packages/frontend/src/components/Modal/ListModal.tsx

```diff
@@ -0,0 +1,71 @@
+import { FunctionComponent, JSXElementConstructor, ReactElement } from 'react';
+
+import Close from '@mui/icons-material/Close';
+import { Box, IconButton, Modal } from '@mui/material';
+
+import { ModalContentWrapper } from '../../styles/common';
+import { HighlightedTitle } from '../Typography/HighlightedTitle';
+import { Colors } from '../../constants/Colors';
+import { useTranslation } from 'react-i18next';
+import { SnackbarAlert } from '../Notification/Snackbar';
+
+interface ModalProps {
+  title?: string;
+  children?: ReactElement<any, string | JSXElementConstructor<any>>;
+  open: boolean;
+  error?: boolean;
+  success?: boolean;
+  handleClose: () => void;
+  onSnackbarClose?: () => void;
+}
+
+export const ListModal: FunctionComponent<ModalProps> = ({
```

Comment:
Is ListModal component duplicated version of FileModal? I can't see their differences yet

Reply:
There are differences in some messages. We could pass those as props but I'm afraid we could create too generic component with too complex logic. 🤷🏻 

Maybe it would make more sense to keep this component as generic modal component and merge those two together.

Reply:
I also have same thinking that should merge these 2 modal components and re-name this component something more generic, e.g. ItemUploadModal, ContentUploadModal or EntityUploadModal which is more generic, could cover both file upload and list title name.